package.json

```diff
@@ -26,6 +26,7 @@
         "clean": "rm -rf frontend/dist && mkdir frontend/dist",
         "build": "yarn copy-scripts && yarn build:esbuild",
         "build:esbuild": "node frontend/build.mjs",
+        "schema:build": "ts-json-schema-generator -f tsconfig.json --path 'frontend/src/*.ts' --type 'AnyNode' --no-type-check > frontend/src/queries/nodes.json && prettier --write frontend/src/queries/nodes.json",
```

Comment:
This is all amazing. A request from me is if we do start to have a bunch of interfaces (interfaces are amazing!) can we standardize a location for them?
something like `/posthog/idl/` or something?